composer.json

```
@@ -58,8 +58,8 @@
 		"league/container": "^4.2",
 		"mikey179/vfsstream": "1.6.11",
 		"mnsami/composer-custom-directory-installer": "^2.0",
-		"mobiledetect/mobiledetectlib": "^2.8",
-		"php-stubs/wordpress-tests-stubs": "^6.5",
+		"mobiledetect/mobiledetectlib": "^2.8 || ^4.8",
+		"php-stubs/wordpress-tests-stubs": "^6.5 || ^6.8",
```

```suggestion
		"php-stubs/wordpress-tests-stubs": "^6.8",
```

`^6.5` already matches with `^6.8` anyway